test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/OnRemoteControlSettings/002_REJECTED_in_case_nonFULL_HMI_level.lua

```diff
@@ -0,0 +1,67 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/10
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/resource_allocation.md
+-- Item: Use Case 1: Exception 1
+--
+-- Requirement summary:
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description: TRS: OnRemoteControlSettings, #2
+-- In case:
+-- 1) SDL received OnRemoteControlSettings notification from HMI with allowed:true
+-- 2) and with any "accessMode" or without "accessMode" parameter at all
+-- 3) and RC_module on HMI is alreay in control by RC-application_1
+-- 4) and another RC_application_2 is in HMILevel other than FULL (either LIMITED or BACKGROUND)
+-- SDL must:
+-- 1) deny access to RC_module for another RC_application_2 after it sends control RPC
+-- (either SetInteriorVehicleData or ButtonPress) for the same RC_module without asking a driver
+-- 2) not process the request from RC_application_2 and respond with result code REJECTED, success:false
+-- 3) leave RC_application_1 in control of the RC_module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--modules array does not contain "RADIO" because "RADIO" module has read only parameters
+local modules = { "CLIMATE", "AUDIO", "LIGHT", "HMI_SETTINGS" }
+local access_modes = { nil, "AUTO_ALLOW", "AUTO_DENY", "ASK_DRIVER" }
```

Comment:
If nil marks the end of a table, this test is broken

Reply:
In these all cases nil value will be used as value and first element of array, and then will be used other values.